packages/jquery/patch/radio-button-hocs.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState, HookContainer } from '@devextreme/runtime/inferno-hooks';
```

Comment:
Is this code written by hand?

Why I'm asking about it:
E.g we change the 'handle change' method in the 'with radio group' HoC in the react component.
Should we remember about this code for jQuery and fix it here too?

Reply:
Yes. I sync it with react version.